examples/jupyter/pincell.ipynb

```diff
@@ -1004,20 +985,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 36,
+   "execution_count": 40,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# OpenMC simulation parameters\n",
+    "batches = 100\n",
+    "inactive = 10\n",
+    "particles = 1000\n",
+    "\n",
+    "# Instantiate a Settings object\n",
     "settings = openmc.Settings()\n",
-    "settings.source = src\n",
-    "settings.batches = 100\n",
-    "settings.inactive = 10\n",
-    "settings.particles = 1000"
+    "settings.batches = batches\n",
+    "settings.inactive = inactive\n",
+    "settings.particles = particles"
```

Comment:
I don't see a good reason to create a separate variable for the batches and particles here. Let's keep it as:
```
settings.batches = 100
...
```

Reply:
@cjwyett It looks like this one still needs updating. After that, we should be good to merge :+1:

Reply:
Apologies for the delay I've been snowed under. I will update pincell and a few more today.

I was also wondering whether you reckon a `from openmc import *` would make everything look a lot neater? Besides, those using openmc are likely already in their openmc environment, and most probably making models which requires a constant barrage of dozens of openmc function calls.  
 
Although, this approach may be less descriptive to those new to Python.

Reply:
`from x import *` is generally not a good idea in Python because it makes it unclear where names in the namespace came from (PEP 8 discourages using it)